aiida/orm/nodes/data/numeric.py

```diff
@@ -124,3 +124,9 @@ def __float__(self):
 
     def __int__(self):
         return int(self.value)
+
+    def __pos__(self):
+        return self.value
+
+    def __neg__(self):
+        return self.new(-self.value)
```

Comment:
Maybe we should also add the `abs` operator which is listed together with `+`, `-` and `~` as the unary operators in the [official Python documentation](https://docs.python.org/3/reference/datamodel.html#emulating-numeric-types). We can omit the `~` invert operator since that doesn't make sense for floats and I doubt anyone would want to use it for `Int`.

```suggestion
        return self.new(-self.value)
```